app/invocation/invocation_suggestion_card.tsx

```diff
@@ -56,43 +43,34 @@ export default class SuggestionCardComponent extends React.Component {
     },
   ];
 
-  componentDidMount() {
-    this.updateSuggestion();
-  }
-
-  componentDidUpdate(prevProps: Props) {
-    if (prevProps.model !== this.props.model) {
-      this.updateSuggestion();
```

Comment:
This was to prevent us from having to run a regency max over the build logs on every render. Can be expensive for large logs.

Reply:
gotcha -- added a `shouldComponentUpdate` so that it doesn't re-render unless new logs come in.